go/mysql/vault/auth_server_vault.go

```diff
@@ -80,7 +51,7 @@ type AuthServerVault struct {
 }
 
 // InitAuthServerVault - entrypoint for initialization of Vault AuthServer implementation
-func InitAuthServerVault() {
+func InitAuthServerVault(vaultAddr string, vaultTimeout time.Duration, vaultCACert, vaultPath string, vaultCacheTTL time.Duration, vaultTokenFile, vaultRoleID, vaultRoleSecretIDFile, vaultRoleMountPoint string) {
```

Comment:
These are quite a few args. Maybe enough that we should construct a struct out of them.

Reply:
I thought about that, but we already use the whole list one call site down as well. So I don't consider it to really be anything significantly worse than before.